src/Orleans/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
-    "Newtonsoft.Json": "7.0.1"
+    "Newtonsoft.Json": "7.0.1",
+    "Microsoft.Extensions.DependencyInjection": "1.0.0"
```

Comment:
What are your thoughts on this? It seems that the .Abstractions package isn't sufficient (eg, doesn't have `GetRequiredService` or `IServiceCollection`)